### PR TITLE
Updated GEM-data with access to results from Built-up Use-Case.

### PR DESCRIPTION
**Pull request by batic**

### CODE REVIEW

File: GEM-data/figs/open_buildings_africa.png
Comment (mlubej) [inline GEM-data/figs/open_buildings_africa.png:1]: there's a hole in the image, is that OK?

File: GEM-data/figs/tfcn_sampling.png
Comment (mlubej) [inline GEM-data/figs/tfcn_sampling.png:1]: I guess it is

File: GEM-data/gem-ML-datacubes.ipynb
Comment (mlubej) [inline GEM-data/gem-ML-datacubes.ipynb:1]: Extra backslash in the markdown cell

File: GEM-data/gem-datasets.ipynb
Comment (mlubej) [inline GEM-data/gem-datasets.ipynb:1]: Links don't work

Comment (batic) [reply GEM-data/gem-datasets.ipynb:1]: Have you tried them in jupyter or online?

Comment (mlubej) [reply GEM-data/gem-datasets.ipynb:1]: opened the notebook online in github

Comment (batic) [reply GEM-data/gem-datasets.ipynb:1]: Yeah, links are really meant to work when you open and use the notebook in jupyter (lab/notebook/app).

**Pull request merged**